molgenis/ds-tidyverse
# feat: case_when

## To test
- See instructions on corresponding client-side pull request [#44](https://github.com/molgenis/ds-tidyverse-client/pull/44)
